src/components/Layouts/ReportLayout.less

```diff
@@ -29,6 +29,11 @@ h1 {
   .report-section {
     display: table-cell;
   }
+  .react-grid-item {
+    overflow: hidden;
+    transition: none;
+    z-index: 1;
```

Comment:
why z-index: 1?

Reply:
always be on top when no z-index is set. Not required tho, mostly for drag and drop